github/ci/prow-deploy/files/jobs/RHsyseng/rhcos-slb/rhcos-slb-presubmits.yaml

```diff
@@ -0,0 +1,35 @@
+---
+presubmits:
+  RHsyseng/rhcos-slb:
+    - name: coreos-assembler-ci-network-tests-main
+      branches:
+        - main
+      annotations:
+        fork-per-release: "true"
+      always_run: false
+      optional: true
+      decorate: true
+      cluster: prow-workloads
+      decoration_config:
+        timeout: 30m
+        grace_period: 5m
+      max_concurrency: 1
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: quay.io/kubevirtci/kubevirt-infra-bootstrap:v20201201-08dc4a9
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
```

Comment:
Does your job really need this much of memory?

Reply:
I don't think so. probably it could do fine with 8Gi, maybe even 4Gi. I will experiment with reducing it once I see that it passes for the first time.

Reply:
@dhiller UPDATE - I [changed](https://github.com/kubevirt/project-infra/compare/2c250d56d6939423a93a395100d04e8e49d43fd7..38a7789f1aa0e09c84395ac4f8f02e35374222b1) the memory to `4Gi`